source/fundamentals/connection/network-compression.txt

```diff
@@ -0,0 +1,97 @@
+.. _node-network-compression:
+
+===================
+Network Compression
+===================
+
+You can enable a driver option to compress messages, which reduces the amount
+of data passed over the network between MongoDB and your application.
+
+The driver supports the following compression algorithms:
+
+1. `Snappy <https://google.github.io/snappy/>`__: available in MongoDB 3.6 and later.
+
+2. `Zlib <https://zlib.net/>`__: available in MongoDB 3.6 and later.
+
+3. `Zstandard <https://github.com/facebook/zstd/>`__: available in MongoDB 4.2 and later.
+
+If you specify multiple compression algorithms, the driver selects the
+first one in the list supported by your MongoDB instance.
+
+.. note::
+
+   When using the Snappy or Zlib compression algorithm, you must
+   :ref:`add explicit dependencies <compression-dependencies>`.
+
+.. _enable-compression:
+
+Specify Compression Algorithms
+------------------------------
+
+You can enable compression for the connection to your MongoDB instance
+by specifying the algorithms in one of two ways: 
+
+1. Adding the parameter to your connection string. 
+2. Specifying the ``compressors`` option in your ``MongoClientOptions``.
+
+.. tabs::
+
+   .. tab:: Connection String
+      :tabid: connection-string
+
+      To enable compression using the connection string, add the
+      ``compressors`` parameter in the connection string. You can
+      specify one or more compression algorithms, separating them with
+      commas:
+
+      .. code-block:: javascript
+         :emphasize-lines: 2
+
+         const uri =
+           "mongodb+srv://<user>:<password>@<cluster-url>/?compressors=snappy,zlib";
+
+         const client = new MongoClient(uri);
+
+   .. tab:: MongoClientOptions
+      :tabid: mongoclientoptions
+
+      To enable compression using the `MongoClientOptions <{+api+}/api/global.html#MongoClientOptions>`__,
+      pass the ``compressors`` option along with the compression
+      algorithm you want to use. You can specify one or more compression
+      algorithms, separating them with commas:
+
+      .. code-block:: javascript
+         :emphasize-lines: 6
+
+         const uri =
+           "mongodb+srv://<user>:<password>@<cluster-url>";
+
+         const client = new MongoClient(uri,
+           {
+             compressors: "snappy"
+           });
+
```

Comment:
It looks like it takes an array of strings if more than one is specified https://mongodb.github.io/node-mongodb-native/4.6/interfaces/MongoClientOptions.html#compressors